recipes/reago/meta.yaml

```diff
@@ -27,5 +27,5 @@ test:
 
 about:
   home: https://github.com/chengyuan/reago-1.1
```

Comment:
I'm still not sure about that part. Conda build counts 4 files, but I'm not sure if it is excluding the setup.py and the MANIFEST.in or not. And I can't tell for sure because it deletes the files locally since the test/build succeeds.